packages/stems/src/components/HarmonyButton/HarmonyButton.module.css

```diff
@@ -23,6 +23,11 @@
   pointer-events: none;
 }
 
+.icon,
+.text {
+  z-index: 1;
+}
```

Comment:
This keeps the text above the `.button::before` overlay so the active state doesn't darken the text.

Not sure this is a good way to accomplish this. Open to suggestions.